frontend/src/components/Pages/RobotPage/LocalizationDialog.tsx

```diff
@@ -1,228 +0,0 @@
-import { Autocomplete, AutocompleteChanges, Button, Card, Dialog, Typography, Icon } from '@equinor/eds-core-react'
-import styled from 'styled-components'
-import { useLanguageContext } from 'components/Contexts/LanguageContext'
-import { Icons } from 'utils/icons'
-import { useState } from 'react'
-import { BackendAPICaller } from 'api/ApiCaller'
-import { Area } from 'models/Area'
-import { Robot } from 'models/Robot'
-import { AreaMapView } from './AreaMapView'
```

Comment:
This is not in use any other places and should also be removed

Reply:
This already removed Edda. Did you maybe make a mistake in loading up changed files otherwise just write on teams =). I am resolving for now,

Reply:
I meant that the AreaMapView function isn't in use anywhere else, so the AreaMapView function within the AreaMapView file can be deleted completely